### ISSUE TITLE
Youtube locking all videos as private. 

### ISSUE THREAD

**Issue by leetwanker** [ryanfox1985/OctoPrint-Youtube-Timelapse#18]
> For videos that have been locked as private due to upload via an unverified API service, you will not be able to appeal. You’ll need to re-upload the video via a verified API service or via the YouTube app/site. The unverified API service can also apply for an API audit.

https://support.google.com/youtube/answer/7300965?hl=en#:~:text=For%20videos%20that%20have%20been,apply%20for%20an%20API%20audit.

**Comment by leetwanker** [comment]
More bad news here, it seems:

https://stackoverflow.com/questions/64079139/using-youtube-data-api-makes-my-videos-private-on-upload

**Comment by NeverminedDE** [comment]
I have the ame issue

**Comment by WimVDK** [comment]
So that's the reason mine keep getting tagged as "violating policy on spam, misleading practices and scams". Guess that makes this plugin already obsolete. Unless we get our "app" reviewed and published?

**Comment by ryanfox1985** [comment]
I don't think it's obsolete, at least you can upload the videos as private.

**Comment by WimVDK** [comment]
The whole point of uploading is to make em (at least for me) public. The moment you currently try to publish the video's that were uploaded as private, youtube pulls them offline again

**Comment by XxbazingaxX** [comment]
Hi, I can't upload the videos to YouTube, I already did all the steps in the guide and gave it permissions to upload videos but when I press the upload I don't see anything upload

**Comment by hryanl** [comment]
> I don't think it's obsolete, at least you can upload the videos as private.

@ryanfox1985, you're right that this plugin is not obsolete if your objective is to upload your OctoLapse videos to your YT account for archival and private viewing. 

However, I think the majority of us who are using this plugin want to make our videos public to share with others. I spent the time to configure this plugin only to find out the videos are locked to private, essentially rendering this plugin obsolete for me.

Please update the description of this plugin to clarify that videos uploaded to YT via this plugin will automatically be set to private if there is an attempt to publish them publicly and they will be locked that way. 

Thanks for you work on this. Hopefully you can rewrite it using some of the example code in the Stack Overflow link referenced earlier.

**Comment by XxbazingaxX** [comment]
I do not say that it is obsolete, only that it does not upload the video (neither private, nor public) there is no video in my account :S De: Ryan LaytonEnviado: domingo, 29 de agosto de 2021 8:25Para: ryanfox1985/OctoPrint-Youtube-TimelapseCC: XxbazingaxX; CommentAsunto: Re: [ryanfox1985/OctoPrint-Youtube-Timelapse] Youtube locking all videos as private. (#18) I don't think it's obsolete, at least you can upload the videos as ***@***.***, you're right that this plugin is not obsolete if your objective is to upload your OctoLapse videos to your YT account for archival and private viewing.However, I think the majority of us who are using this plugin want to make our videos public to share with others. I spent the time to configure this plugin only to find out the videos are locked to private, essentially rendering this plugin obsolete for me.Please update the description of this plugin to clarify that videos uploaded to YT via this plugin will automatically be set to private if there is an attempt to publish them publicly and they will be locked that way.Thanks for you work on this. Hopefully you can rewrite it using some of the example code in the Stack Overflow link referenced earlier.—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
